modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -59,10 +59,9 @@ there are several self-managed options available:
 Couchbase Capella::
 +
 --
-If you haven't already got a cluster set up, the easiest route is to https://cloud.couchbase.com/sign-up[sign up to Couchbase Capella and deploy a free tier cluster^], 
-then come back to this page.
-Make a note of the xref:cloud:get-started:connect.adoc[endpoint] to connect to, 
-and remember the credentials for the user that you set up.
+If you haven't already got a cluster set up, the easiest route is to https://cloud.couchbase.com/sign-up[sign up to Couchbase Capella and deploy a free tier cluster^]. 
+Once the set up is complete, come back to this page.
+Make a note of the xref:cloud:get-started:connect.adoc[endpoint] to connect to, and remember the credentials for the user that you set up.
```

Comment:
These are semi-independent clauses,
and quite happily live on separate line.

Our one-sentence-per line is aimed more at stopping people cramming multiple sentences into a line,
or preventing 80-char automatic breaks,
than stopping clauses living separately.

the overall goal should be readability for docs editors - both of source text and of diffs.